new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItSimpleOperatorValidation.java

```diff
@@ -65,69 +64,89 @@ public void testInstallingOperator() {
     // the kubernetes deployment.  this will complete quickly, and will either be
     // successful or not.
 
-    // get a new unique namespace
-    final String namespace = assertDoesNotThrow(TestActions::createUniqueNamespace,
+    // get a new unique opNamespace
+    final String opNamespace = assertDoesNotThrow(TestActions::createUniqueNamespace,
+        "Failed to create unique namespace due to ApiException");
+    logger.info(String.format("Got a new namespace called %s", opNamespace));
+
+    final String domainNamespace1 = assertDoesNotThrow(TestActions::createUniqueNamespace,
+        "Failed to create unique namespace due to ApiException");
+    logger.info(String.format("Got a new namespace called %s", domainNamespace1));
+
+    final String domainNamespace2 = assertDoesNotThrow(TestActions::createUniqueNamespace,
         "Failed to create unique namespace due to ApiException");
-    logger.info(String.format("Got a new namespace called %s", namespace));
+    logger.info(String.format("Got a new namespace called %s", domainNamespace2));
```

Comment:
-> "Created a new namespace called ..."
Doesn't the logger facade in the operator support String formats like "blah blah {0}"?  Shouldn't we use the logging facade in operator rather than java.util.logging directly?

Reply:
this is the logger injected by `LoggedTest` isn't it?

Reply:
@ddsharpe will change the message. For logger, it should be fixed in LoggedTest as Mark said.

Reply:
changed the message